Digipost.Signature.Api.Client.Core/Internal/Asice/AsiceSignature/SignedXmlWithAgnosticId.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography.Xml;
+using System.Xml;
+using Difi.Felles.Utility.Exceptions;
+using Digipost.Signature.Api.Client.Core.Exceptions;
+
+namespace Digipost.Signature.Api.Client.Core.Internal.Asice.AsiceSignature
+{
+    /// <summary>
+    ///     Enhances the core SignedXml provider with namespace agnostic query for Id elements.
+    /// </summary>
+    /// <remarks>
+    ///     From:
+    ///     http://stackoverflow.com/questions/5099156/malformed-reference-element-when-adding-a-reference-based-on-an-id-attribute-w
+    /// </remarks>
+    public sealed class SignedXmlWithAgnosticId : SignedXml
```

Comment:
Denne klassen er litt ullen og komplisert og jeg tør ikke fikle for mye her, men en MVP for å rydde opp litt her er å gi engelske feilmeldinger.